lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -131,24 +72,24 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_DAYS, M
    * @function
    * @public
    *
+   * @param {String} version - version
    * @returns {Promise}
    *
    * @example
-   * UpdateNotifierService.notify();
+   * UpdateNotifierService.notify('1.0.0-beta.16');
    */
-  this.notify = () => {
-    return this.getLatestVersion().then((version) => {
-      return ModalService.open({
-        template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
-        controller: 'UpdateNotifierController as modal',
-        size: 'update-notifier',
-        resolve: {
-          options: _.constant({
-            version
-          })
-        }
-      }).result;
-    });
+  this.notify = (version) => {
+    return ModalService.open({
+      template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
+      controller: 'UpdateNotifierController as modal',
+      size: 'update-notifier',
+      resolve: {
+        options: _.constant({
+          version,
+          allowsSleep: shouldAllowSleep(release.getReleaseType(version))
```

Comment:
This also feels a bit odd - surely it's the release-type of the currently running Etcher version that ought to be used to determine whether the update can be 'sleeped', rather than the latest-version available from S3?